third_party/clang-format/git-clang-format

```diff
@@ -178,7 +179,7 @@ def main():
       print('changed files:')
       for filename in changed_files:
         print('    %s' % filename)
-
+  sys.exit(1)
```

Comment:
I'm not sure if this is a right way.
For example, if you change this, it seems to work well but it might not work in the following case.
`$ ./bacardi format && echo "Code formatted successfully!"`

If someone writes above wrapper script, it will not work well.

Reply:
Good point.